src/sparseml/export/helpers.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+from enum import Enum
+from pathlib import Path
+from typing import Callable, List, Union
+
+import onnx
+
+from sparsezoo.utils.onnx import save_onnx
+
+
+__all__ = ["apply_optimizations"]
+
+
+class GraphOptimizationOptions(Enum):
```

Comment:
what's the idea behind this? why an enum instead of bool for now? if we think specific integrations will have different options, then this should likely be handled by kwargs

Reply:
Let's talk about this. When designing this functionality I was planning ahead for more complex use cases, such as the `transformer` export. Specifically for the case, where we allow the user to specify a set of optimizations - thin. This particular Enum is here, to implement the UI consistent with the requirements from Mark.

![Uploading image.png…]()